discord/member.py

```diff
@@ -405,6 +405,9 @@ def activity(self):
         """Union[:class:`BaseActivity`, :class:`Spotify`]: Returns the primary
         activity the user is currently doing. Could be ``None`` if no activity is being done.
 
+        .. note::
+            Due to a Discord API limitation, this may be ``None`` if the user is listening to a song on spotify with a long title.
```

Comment:
```suggestion
        .. note::
        
            Due to a Discord API limitation, this may be ``None`` if the user is listening to a song on spotify with a long title.
```

Reply:
Ah whoops, I'd meant to add a blank line there but must've missed it.